lib/plugins/bull-mq-metrics/MetricsCollector.ts

```diff
@@ -79,6 +73,30 @@ export class MetricsCollector {
     }
   }
 
+  private async discoverQueues(): Promise<ObservableQueue[]> {
```

Comment:
Can't we make a composite queue discoverer that incapsulates all this logic instead of leaking it to the metrics collector?

Reply:
After update I enforced `redisClients` parameter to be provided as an array, and observers also accept arrays now, returning `{ redisInstance, queueName }[]`
It's a breaking change so I will create follow up PR to adjust Polyglot to the new types. 
Let me know if you disagree with that solution